test/core/test/cli-test.test.ts

```diff
@@ -411,6 +411,24 @@ test('public parseCLI works correctly', () => {
     },
   })
 
+  expect(parseCLI('vitest --project="space 1"')).toEqual({
+    filter: [],
+    options: {
+      'project': ['space 1'],
+      '--': [],
+      'color': true,
+    },
+  })
+
+  expect(parseCLI('vitest --project="space 1" --project="space 2"')).toEqual({
```

Comment:
can you add a test for multiple filters? Like `vitest ./test-1.js ./test-2.js --some-args`

Reply:
Done.